packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.ts

```diff
@@ -175,6 +176,45 @@ export async function injectRecaptchaFields<T>(
   return newRequest;
 }
 
+type ActionMethod<TRequest, TResponse> = (
+  auth: Auth,
+  request: TRequest
+) => Promise<TResponse>;
+
+export async function handleRecaptchaFlow<TRequest, TResponse>(
```

Comment:
can we add some tests for this method? I understand that usages of this methods in the various flows are being tested.. would be good to test this too.

Reply:
@renkelvin can you update the description that all rCE protected flows were manually verified with the demo app?

I am ok with moving the new unit tests to a different PR to debug them.

Reply:
Yeah, done.